esp8266/modules/webrepl.py

```diff
@@ -59,18 +59,16 @@ def start(port=8266, password=None):
     stop()
     if password is None:
         try:
-            import port_config
-            _webrepl.password(port_config.WEBREPL_PASS)
+            import webrepl_cfg
+            _webrepl.password(webrepl_cfg.PASS)
```

Comment:
Why does it have to be a python file? Why not just use a simple text file, with nothing in it but the password? Then you don't need to weird tricks trying to "parse" the python code later.

Even better, save the password in the boot.py, just always passing it as the argument to start. This way you won't need half of this code.